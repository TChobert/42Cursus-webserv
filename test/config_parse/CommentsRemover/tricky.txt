value=12345#6
name=#john
# This comment starts with a hash
notacomment=###notacomment
