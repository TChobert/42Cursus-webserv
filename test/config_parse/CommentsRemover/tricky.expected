value=12345
name=
notacomment=
